kubernetes/cmsweb/hpa/dbs-hpa.yaml

```diff
@@ -8,8 +8,8 @@ spec:
     apiVersion: apps/v1
     kind: Deployment
     name: dbs-global-r # should match deployment target
-  minReplicas: 8
-  maxReplicas: 10
+  minReplicas: 5
```

Comment:
@muhammadimranfarooqi , @yuyiguo , I thought for DBS global-r we need range 8-10, and not 5-8. Please double check before I need to merge this PR.

@yuyiguo , please confirm that these changes reflect what is necessary for DBS.

Reply:
@vkuznet @muhammadimranfarooqi 

It should be 8 to 10.  

Are you on vacation, Valentin? :-)